src/universal/modules/welcome/containers/Welcome/Welcome.js

```diff
@@ -3,26 +3,52 @@ import {connect} from 'react-redux';
 import {formValueSelector} from 'redux-form';
 import {HotKeys} from 'react-hotkeys';
 import requireAuth from 'universal/decorators/requireAuth/requireAuth';
+import {authedOptions} from 'universal/redux/getAuthedUser';
+
+import {mutationHandlers as Step2MutationHandlers} from '../../components/WelcomeWizardForms/Step2TeamName';
 
 import {
   Step1PreferredName,
   Step2TeamName,
   Step3InviteTeam
 } from '../../components/WelcomeWizardForms';
 
+/*
+ * Setup and extend requireAuth cashay mutation handlers:
+ */
+const cashayAuthQueryOpts = {
+  component: 'WelcomeContainer',
+  mutationHandlers: Object.assign(authedOptions.mutationHandlers,
```

Comment:
Nah, this is one of those patterns I was hoping would emerge-- you have a file full of shared `mutationHandlers` & each component that uses the same query borrows from that. Only thing I'd change is not mutating the object coming from `authedOptions`: `Object.assign({}, authedOptions.mutationHandlers, ...`.